api/v1alpha1/flowcollector_types.go

```diff
@@ -32,7 +32,10 @@ const (
 
 func (spec *FlowCollectorSpec) UseEBPF() bool  { return spec.Agent.Type == AgentEBPF }
 func (spec *FlowCollectorSpec) UseIPFIX() bool { return spec.Agent.Type == AgentIPFIX }
-func (spec *FlowCollectorSpec) UseKafka() bool { return spec.DeploymentModel == DeploymentModelKafka }
+func (spec *FlowCollectorSpec) UseKafka() bool {
```

Comment:
I'm not sure about this ... does it mean that the user can set "DeploymentModel" to "DIRECT" and still use Kafka? I don't think it should be allowed

Reply:
sure, let's try this approach for first version